src/BurgerIcon.js

```diff
@@ -40,11 +40,23 @@ let BurgerIcon = React.createClass({
       outline: 'none'
     });
 
+    var containerStyle = appendVendorPrefix({
+      position: "absolute",
+      left: "10%",
+      right: "10%",
+      top: "10%",
+      bottom : "10%",
+      margin: 0,
+      padding: 0
+    });
+
     return (
-      <div>
-        <span className="bm-burger-icon" style={ this.getLineStyle(0) }></span>
-        <span className="bm-burger-icon" style={ this.getLineStyle(1) }></span>
-        <span className="bm-burger-icon" style={ this.getLineStyle(2) }></span>
+      <div className="bm-burger-button">
+        <div className="bm-burger-icon-container" style={ containerStyle }>
```

Comment:
Alright, that make sense. I added it to 'simulate' padding in a layout based on relative positioning (to avoid complicated calculations for the bars themselves), but if you're fine with a full-size visual icon, then it's not needed.